pkg/gofr/testutil/port_test.go

```diff
@@ -19,3 +20,16 @@ func TestGetFreePort(t *testing.T) {
 
 	_ = listener.Close()
 }
+
+func TestServerConfigsProvider(t *testing.T) {
+	env := ServerConfigsProvider(t)
```

Comment:
Please add an asserter for the the env variables. Something like a check on the fact the env variables are set and not empty

You can also check for equality